### PR TITLE
Test parallelization

### PR DESCRIPTION
**Pull request by Eiriknil**

### CODE REVIEW

File: buildspec.test_parallel.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 0.2
+
+batch:
+  fast-fail: false
+  build-list:
+    - identifier: test1
```
Comment (brinxmat) [inline buildspec.test_parallel.yml:6]: - identifier: TestParallizationRunner1

File: buildspec.test_parallel.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      buildspec: buildspec.testrun.yml
+      env:
+        variables:
+          BUILD_ID: test1
+    - identifier: test2
```
Comment (brinxmat) [inline buildspec.test_parallel.yml:11]: Ditto

File: buildspec.testdata.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+phases:
+  install:
+    runtime-versions:
+      python: 3.x
+  pre_build:
+    commands:
+      - wget https://github.com/BIBSYSDEV/NVA-end-to-end-testing/archive/refs/heads/test-deploy.zip
+      - unzip test-deploy.zip
+      - cd NVA-end-to-end-testing-test-deploy
+      - npm install
```
Comment (brinxmat) [inline buildspec.testdata.yml:20]: Unneeded?

File: buildspec.testrun.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - identifier: cypress-e2e-tests
+      env:
+        variables:
+          image: public.ecr.aws/cypress-io/cypress/browsers:node14.17.0-chrome88-ff89
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
+      python: 3.x
```
Comment (brinxmat) [inline buildspec.testrun.yml:25]: Remove?

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: buildspec.test_parallel.yml [hunk 5]
```diff
@@ -1,16 +1,24 @@
...
+#   env:
+#     variables:
+#       BUILD_ID: <build_id>
+
+
 batch:
   fast-fail: false
   build-list:
-    - identifier: test1
+    - identifier: TestParallizationRunner1
```
Comment (brinxmat) [inline buildspec.test_parallel.yml:14]: Parallelization

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: buildspec.test_parallel.yml (hunk 5 above)
Comment (Eiriknil) [reply buildspec.test_parallel.yml:14]: Vanskelig ord...

**Review by brinxmat — APPROVED** [review]

**Pull request merged**